app/controllers/v1/user.js

```diff
@@ -1,5 +1,6 @@
 "use strict";
 
+const {default: logger} = require('@financial-times/n-logger');
```

Comment:
I prefer spaces at start/end whiled deconstruct. IMO it simplifies readability, feel free to ignore. I'm not sure this is consistent with the style of the rest of the code in the repo.
```suggestion
const { default: logger } = require('@financial-times/n-logger');
```
BTW, I don't see any usage of this var here...am I missing something or this is useless?

Reply:
good spot. I left it in by mistake

Reply:
I agree, however there are many stylistic issues in this repository anyway, this falls way down in priority. I may fix in a later PR

Reply:
@leggsimon It's up to you, so feel free to ignore. All of them will probably take 1 minute to be fixed and I doubt you will have ever the time to go back and fix this stuff on another occasion :)